oxidation/libparsec/crates/protocol/schema/authenticated_cmds/pki_enrollment_accept.json

```diff
@@ -0,0 +1,184 @@
+[
+    {
+        "label": "PkiEnrollmentAccept",
+        "major_versions": [
+            3
+        ],
+        "introduced_in": "3.2",
+        "req": {
+            "cmd": "pki_enrollment_accept",
+            "fields": {
+                // Signature should be checked before loading.
+                "accept_payload": {
+                    "type": "Bytes"
+                },
+                "accept_payload_signature": {
+                    "type": "Bytes"
+                },
+                "accepter_der_x509_certificate": {
+                    "type": "Bytes"
+                },
+                "enrollment_id": {
+                    "type": "UUID"
+                },
+                "device_certificate": {
+                    "type": "Bytes"
+                },
+                "user_certificate": {
+                    "type": "Bytes"
+                },
+                // Same certificates than above, but expunged of human_handle/device_label
+                "redacted_device_certificate": {
+                    "type": "Bytes"
+                },
+                "redacted_user_certificate": {
+                    "type": "Bytes"
+                }
+            }
+        },
+        "reps": {
+            "ok": {},
+            "not_allowed": {
+                "fields": {
+                    "reason": {
+                        "type": "String"
```

Comment:
Why reason is mandatory ?

Reply:
The reason seems to be allways provided :shrug:

https://github.com/Scille/parsec-cloud/blob/d057522e581d7d637d2999830990033a45d5cd2e/parsec/backend/pki.py#L274-L277

Reply:
https://github.com/Scille/parsec-cloud/blob/9ee327b2d30643491a3988b01b66de944a3bb607/parsec/api/protocol/base.py#L102-L106

`reason` field is not mandatory, and we are planning to remove it entirely in the future ;-)

Reply:
If found gold, but the response serializer for `pki_enrollment_*` don't inherit this class `ErrorRepSchema`. That's a miss ?

Reply:
Anyway, I've update the definition